src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(), lottery.getRandomBall(), lottery.getRandomBall()};
+        for (Ball item : balls) {
+            System.out.println(item.toString());
```

Comment:
toString is not necessary to call in sout, it is called implicitly anyway
```suggestion
            System.out.println(item);
```